lib/Agrammon/Web/Routes.pm6

```diff
@@ -186,10 +186,10 @@ sub dataset-routes(Agrammon::Web::Service $ws) {
 sub user-routes(Agrammon::Web::Service $ws) {
     route {
 
-        # test
+        # working
         post -> LoggedIn $user, 'change_password' {
             request-body -> (:oldPassword($old-password)!, :newPassword($new-password)!) {
-                my $data = $ws.change-password($old-password, $new-password);
+                my $data = $ws.change-password($user, $old-password, $new-password);
                 content 'application/json', $data;
```

Comment:
When map it into something nice for the response here.

Reply:
Also, should a different HTTP status be used to indicate the error situation, rather than a boolean?

Reply:
But at least it should return the same "shape" of data if a boolean is being used for it.

Reply:
Completely reworked.